app/models/comment.rb

```diff
@@ -77,13 +77,25 @@ def mentioned_users
     User.find_all_by_username(usernames.map {|m| m[1] }).uniq
   end
 
+  def mentioned_tag_followers
```

Comment:
Here, is this followers of the tag who were /also/ mentioned? I think mentions should notify people regardless of whether they're following the tag, so this may need to be reworked and the method name adjusted to match. Thank you, though, your code is very readable!